src/nxdoc/nuxeo-add-ons/nuxeo-management-rest-api.md

```diff
@@ -0,0 +1,762 @@
+   "completed": null,
+   "processingMillis": 0
+}
+```
+
+### Distribution Endpoint
+
+To get the distribution information.
+
+```
+GET https://localhost:8080/nuxeo/site/management/distribution
+```
+
+**Response**
+
+Returns a JSON representation of the distribution information.
+
+**Status Codes**
+
+- 200 *OK* - Success.
+
+**Sample**
+
+```curl
+curl -u Administrator:Administrator \
+https://localhost:8080/nuxeo/site/management/distribution
+```
+
+```json
+{
+  "org.nuxeo.ecm.product.name": "Nuxeo Platform",
+  "org.nuxeo.ecm.product.version": "LTS 2019-HF17",
+  "org.nuxeo.distribution.name": "server",
+  "org.nuxeo.distribution.version": "10.10",
+  "org.nuxeo.distribution.server": "tomcat",
+  "org.nuxeo.distribution.date": "201901211253",
+  "warnings": [],
+  "nuxeo.osgi.bundles": [
+    {
+      "name": "org.nuxeo.admin.center",
+      "version": "10.10-HF11"
+    },
+
+    ...other happy bundles...
+
+  ]
+}
+```
+
+### Elasticsearch Endpoints
+
+The following endpoints are useful when the Nuxeo Platform leverages [Elasticsearch]({{page page='elasticsearch-setup'}}).
+
+#### Reindex
+
+##### Reindex a Repository or a Set of Documents
+
+To reindex an entire [repository]({{page page='elasticsearch-setup'}}#configuration-for-multi-repositories) or a set of documents using [NXQL]({{page page='nxql'}}).
+
+```
+POST https://localhost:8080/nuxeo/site/management/elasticsearch/{REPOSITORY_NAME}/reindex?query=QUERY
+```
+
+**Path Parameters**
+
+| Parameter Name        | Type       | Required | Description          |
+| --------------------- | ---------- | -------- | -------------------- |
+| **REPOSITORY_NAME**   | **string** |    Yes   | The repository name. |
+
+**Query Parameters**
+
+| Parameter Name        | Type       | Required | Description                     |
+| --------------------- | ---------- | -------- | ------------------------------- |
+| **query**             | **string** |    No    | The NXQL query.                 |
+
+**Response**
+
+If successful, returns a JSON representation (see below) of a [Bulk Action Status]({{page page='bulk-action-framework'}}#bulk-rest-api).
+
+**Status Codes**
+
+- 200 *OK* - Success.
+
+**Sample**
+
+To reindex the `default` repository:
+
+```curl
+curl -X POST -u Administrator:Administrator \
+https://localhost:8080/nuxeo/site/management/elasticsearch/default/reindex
+```
+
+```json
+{
+   "entity-type": "bulkStatus",
+   "commandId": "d3bc0a81-b061-447f-b307-c3db78b5f457",
+   "state": "SCHEDULED",
+   "processed": 0,
+   "error": false,
+   "errorCount": 0,
+   "total": 0,
+   "action": "index",
+   "username": "system",
+   "submitted": "2019-07-26T14:12:29.224Z",
+   "scrollStart": null,
+   "scrollEnd": null,
+   "processingStart": null,
+   "processingEnd": null,
+   "completed": null,
+   "processingMillis": 0
+}
+```
+
+To reindex a set of documents on a given Nuxeo repository matching the [NXQL]({{page page='nxql'}}) query `select * from document where dc:title like 'My Title%'`:
```

Comment:
To follow the NXQL documentation, reformat the query to: `SELECT * FROM Document WHERE dc:title LIKE 'My Title%'`.
Same in the `curl` command.